esx_service/vmdk_ops.py

```diff
@@ -342,51 +347,109 @@ def detachVMDK(vmdk_path, vm_uuid):
     return disk_detach(vmdk_path, vm)
 
 
-# Check existence (and creates if needed) the path
-def getVolPath(vm_config_path):
-    # The folder for Docker volumes is created on <vm_datastore>/DOCK_VOLS_DIR
-    # On ESX the path is always /vmfs/volume/datastore/... , so we can do this:
-    vm_datastore = "/".join(vm_config_path.split("/")[0:4])
-    path = os.path.join(vm_datastore, DOCK_VOLS_DIR)
-
+# Check existence (and creates if needed) the path for docker volume VMDKs
+def get_vol_path(datastore):
+    # The folder for Docker volumes is created on <datastore>/DOCK_VOLS_DIR
+    path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
+    
     if os.path.isdir(path):
         # If the path exists then return it as is.
         logging.debug("Found %s, returning", path)
         return path
-
+    
```

Comment:
lack of these spaces screws up python auto-formatting in editor and external tools....